### ISSUE TITLE
hurricane survivors

### ISSUE THREAD

**Issue by randy7771026** [Irma-Response/irmarelief-static#7]
Just memories that Katrina folks who came here preferred that

**Comment by miklb** [comment]
@randy7771026 I agree. Webflow doesn't allow the renaming of slugs so the URL fragment couldn't be  changed.
